src/telemetry/connectionTelemetry.ts

```diff
@@ -14,38 +15,40 @@ const { version } = require('../../package.json');
 export type NewConnectionTelemetryEventProperties = {
   auth_strategy?: string;
   is_atlas?: boolean;
-  atlas_host_id?: string | null;
-  is_localhost?: boolean;
+  is_local_atlas?: boolean;
+  atlas_hostname?: string | null;
   is_data_lake?: boolean;
   is_enterprise?: boolean;
-  is_public_cloud?: boolean;
   dl_version?: string | null;
-  public_cloud_name?: string | null;
   is_genuine?: boolean;
   non_genuine_server_name?: string | null;
   server_version?: string;
   server_arch?: string;
-  server_os?: string;
+  server_os_family?: string;
   is_used_connect_screen?: boolean;
   is_used_command_palette?: boolean;
   is_used_saved_connection?: boolean;
   vscode_mdb_extension_version?: string;
-};
+  topology_type?: TopologyType;
+} & HostInformation;
 
-type CloudInfo = {
-  isPublicCloud?: boolean;
-  publicCloudName?: string | null;
+export type HostInformation = {
+  is_localhost?: boolean;
+  is_atlas_url?: boolean;
+  is_do_url?: boolean;
```

Comment:
Readability for folks who might not know `do` = `digital ocean`
```suggestion
  is_do_url?: boolean; // Is digital ocean url.
```